File: mvpLibrary/src/main/java/cn/cbsd/mvplibrary/CommonConfig.java
package cn.cbsd.mvplibrary;


import cn.cbsd.mvplibrary.router.Router;

/**
 * Created by wanglei on 2016/12/4.
 * 全局设置
 */

public class CommonConfig {
    // #log
    public static final boolean LOG = BuildConfig.DEBUG;
    public static final String LOG_TAG = "XLog";

    // #cache
    public static final String CACHE_SP_NAME = "config";
    public static final String CACHE_DISK_DIR = "cache";

    // #router
    public static final int ROUTER_ANIM_ENTER = Router.RES_NONE;
    public static final int ROUTER_ANIM_EXIT = Router.RES_NONE;

    // #imageloader

    // #dev model
    public static final boolean DEV = true;

    public static class Pattern {
        public static final String REG_PHONE = "^1\\d{10}$";
        public static final String REG_KDG_SIZE = "^\\d+\\*\\d+$";
        public static final String REG_PASSWORD = "^(?![0-9]+$)(?![a-zA-Z]+$)(?![<>@$#%^&*()-+!?]+$)[\\w<>@$#%^&*()-+!?]{6,20}$";
        public static final String DEFAULT_DATE = "yyyyMMdd";
        public static final String DEFAULT_DATE_TIME = "yyyyMMdd HH:mm";
        public static final String DEFAULT_TIME = "HH:mm:ss";
    }
}

File: mvpLibrary/src/main/java/cn/cbsd/mvplibrary/CommonConfig.kt
package cn.cbsd.mvplibrary


import cn.cbsd.mvplibrary.router.Router

/**
 * Created by wanglei on 2016/12/4.
 * 全局设置
 */

object CommonConfig {
    // #log
    val LOG = BuildConfig.DEBUG
    val LOG_TAG = "XLog"

    // #cache
    val CACHE_SP_NAME = "config"
    val CACHE_DISK_DIR = "cache"

    // #router
    val ROUTER_ANIM_ENTER = Router.RES_NONE
    val ROUTER_ANIM_EXIT = Router.RES_NONE

    // #imageloader

    // #dev model
    val DEV = true

    object Pattern {
        val REG_PHONE = "^1\\d{10}$"
        val REG_KDG_SIZE = "^\\d+\\*\\d+$"
        val REG_PASSWORD = "^(?![0-9]+$)(?![a-zA-Z]+$)(?![<>@$#%^&*()-+!?]+$)[\\w<>@$#%^&*()-+!?]{6,20}$"
        val DEFAULT_DATE = "yyyyMMdd"
        val DEFAULT_DATE_TIME = "yyyyMMdd HH:mm"
        val DEFAULT_TIME = "HH:mm:ss"
    }
}
